src/scrabble.js

```diff
@@ -1,6 +1,49 @@
+const letterScores = {
+  onePoint: ['a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U', 'l', 'L', 'n', 'N', 'r', 'R', 's', 'S', 't','T'],
+  twoPoints: ['d', 'D', 'g', 'G'],
+  threePoints: ['b', 'B', 'c', 'C', 'm', 'M', 'p', 'P'],
+  fourPoints: ['f', 'F', 'h', 'H', 'v', 'V', 'w', 'W', 'y', 'Y'],
+  fivePoints: ['k', 'K'],
+  eightPoints: ['j', 'J', 'x', 'X'],
+  tenPoints: ['q', 'Q', 'z', 'Z']
+}
+
 class Scrabble {
-  // Write your implementation here
+  constructor(word) {
+    this.word = word
+  }
+  score () {
+    let count = 0
+    if (this.word === null || this.word.length === 0 || this.word === " \t\n") {
+      return count
+    }
+    for (let i = 0; i < this.word.length; i++) {
```

Comment:
in here you could then handle lower case words like:
`let word = this.word.trim().toUpperCase();`